index.html

```diff
@@ -347,9 +151,70 @@ <h3>Skills</h3>
       <img id="indicator" src="assests/images/shape.png" alt="Home Indicator/Light">
     </div>
   </footer>
-
+  
+    <!-- Popup Window -->
+    <div class="modal-overlay">
+      <div class="modal-container">
+          <div class="modal-header">
+            <div class="heading">
+              <span class="close-btn">&times;</span>
+              <p class="mobile-heading">Multi Post Stories</p>
+              <p class="desktop-heading">Keeping track of hundreds of components website</p>
+            </div>
+            <div class="modal-tags">
+              <ul>
+                  <li class="modal-tag">Html</li>
+                  <li class="modal-tag">Bootstrap</li>
+                  <li class="modal-tag">Ruby on Rails</li>
+              </ul>
+            </div>
+          </div>
+  
+          <div class="modal-body">
+            <div class="modal-mobImage">
+               <img class="modal-image" src="assests/images/mobileImg.png" alt="Modal mobile image">
+            </div>
+            <div class="modal-desktopImage">
+               <img class="modal-image" src="assests/images/desktopImg.png" alt="Modal desktop image">
+            </div>
+  
+            <div class="modal-text">
+              <p class="modal-mobileDesc">
+                Lorem Ipsum is simply dummy text of the printing 
+                and typesetting industry. Lorem Ipsum has been 
+                the industry's standard dummy text ever since the 
+                1500s, when an unknown printer took a galley of type 
+                and scrambled it to make a type specimen book. It has 
+                survived not only five centuries, but also the leap into 
+                electronic typesetting, remaining essent
+              </p>
+              <p class="modal-desktopDesc">
+                Lorem Ipsum is simply dummy text of the printing and typesetting 
+                industry. Lorem Ipsum has been the industry's standard dummy text 
+                ever since the 1500s, when an unknown printer took a galley of type 
+                and scrambled it 1960s with the releaLorem
+                Ipsum is simply dummy text of the printing and typesetting 
+                industry. Lorem Ipsum has been the industry's standard dummy text 
+                ever since the 1500s, when an unknown printer took a galley of type 
+                and scrambled it 1960s with the releaLorem
+              </p>
+              <div class="modal-btns">
+                <button>
+                    <a class="modal-btn" href="">See Live</a>
+                    <img class="modal-btn-img" src="assests/images/live-icon.png" alt="See Live Image">
+                </button>
+                <button>
+                    <a class="modal-btn" href="">See Source</a>
+                    <img class="modal-btn-img" src="assests/images/github-icon.png" alt="See Source Github Image">
+                </button>
```

Comment:
- [ ] Kindly implement the correct button styling for the on hover state. This will ensure that your design matches the chosen Figma design.

**These are the desired designs from Figma:**

![image](https://user-images.githubusercontent.com/86318284/203614194-b7c610ad-5239-4962-b663-0036b04d195d.png)

**This is how I am seeing your popup buttons on hover:**

[Screencast from 23-11-2022 08:45:07 ALASIRI.webm](https://user-images.githubusercontent.com/86318284/203614331-736d7711-5060-4998-aab5-7b5cab0ed5b8.webm)